src/views/Publish.vue

```diff
@@ -150,6 +159,55 @@ export default defineComponent({
       apiVersion: RequestsUtils.confAPIVersion,
       // loading indicator
       isPublishLoading: false,
+
+      // Publish History
+      columns: [
+        {
+          title: 'Full Date',
```

Comment:
Title can be just `Date`
```suggestion
          title: 'Date',
```